query/elb/elb_application_drop_http_headers.sql

```diff
@@ -0,0 +1,18 @@
+select
+  -- Required Columns
+  arn as resource,
+  case
+    when region in ('af-south-1', 'ap-northeast-3', 'eu-south-1') then 'skip'
+    when load_balancer_attributes @> '[{"Key": "routing.http.drop_invalid_header_fields.enabled", "Value": "true"}]' then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when region in ('af-south-1', 'ap-northeast-3', 'eu-south-1') then title || ' excluded due to unsupported control region.'
```

Comment:
Do we need this skip? Are ALBs not creatable in these regions, or can we create them but not configure this specific setting? If we can't create ALBs in these regions, we don't need this check at all, else we can keep it in.

Reply:
@cbruno10 I am able to create ALB in ap-northeast-3 & also able to set the attribute. I think the doc is not updated. should I remove this entire check?